src/jdk.jartool/share/classes/jdk/security/jarsigner/JarSigner.java

```diff
@@ -943,6 +949,12 @@ private void writeEntry(ZipFile zf, ZipOutputStream os, ZipEntry ze)
         ze2.setTime(ze.getTime());
         ze2.setComment(ze.getComment());
         ze2.setExtra(ze.getExtra());
+        int perms = JUZFA.getPosixPerms(ze);
+        if (!posixPermsDetected && perms != -1) {
+            posixPermsDetected = true;
+            Event.report(Event.ReporterCategory.POSIXPERMS, "true");
```

Comment:
The method signature is `report(category, type, values...)`. There is no need to define a type or value here but I feel like more normal to call `report(POSIXPERMS, "detected")`.